### DISCUSSION TITLE
Cost basis wrong ?

### DISCUSSION THREAD

**Discussion by vPackets** [hickeng/financial#155] [Q&A]
Hi There!
First I want to thank you for everything you did, that's very helpful..... The amount of work and research is crazy and I salute and thank you for that.

I do not know what I am doing wrong but in the spreadsheet I have checked all the ESPP and RSU granted + their granting date.

For my ESPP: The cost basis is always 0 ? 

![Screenshot 2024-04-16 at 9 35 54 AM](https://github.com/hickeng/financial/assets/17295930/2e9d48d3-f02c-478a-9988-5e6fbef6cab3)

For my RSU the cost basis is quite high and is the results of proceeds (d) - 1099-B Proceeds.

But for example: 

I got granted 203 VMW on 06/01/2021 at 161.26 Market value per share

The spreadsheet shows the following:

1099-B Proceed : $13,860.16
Proceed (d): $39,959.33 
1099-B Cost Basis: $29,370.04

If I check my cost basis for all my modest RSU assigned and apply them in turbo tax ... the refund from the IRS is going substantially HIGH. 

I just want to make sure I understand what I am doing.

Thanks a ton !

**Comment by hickeng** [comment]
@vPackets 

For ESPP the basis is always supposed to be `0.00` in Form8949. For covered securities you must report the same basis reported to the IRS via 1099. Instead, basis correction for ESPP is done via the `Adjustment` column.

If you set the `Display of Form 8949` in `Summary!K36` to `Cash Only` then you'll see the Adjustment is `0.00` for all ESPP lots back to 2014 (I don't have lots before that).

This is because none of the ESPP lots had a tax basis high enough to use the alternate gain calculation (over $128.60 after dividend adjustments). The usage instructions have a section on how to enter this info into Turbotax - it should be similar for other tax prep software.

The Gain column should remain the same, regardless of whether you have `Cash Only` or `Cash & AVGO FMV` selected for display. The Gain value is the bit that dictates the captial gain/(loss) that you pay tax on.

Those two ESPP lots show a captial _gain_ of `9353.9`, so you should be paying tax on that gain. If that results in a refund then you must have substantially overpaid estimated taxes. Personally I've got a massive refund from CA and owe Fed monies - which makes sense for me as I aimed to meet the Fed and State safe harbor thresholds for estimated payments.
